x-pack/plugins/maps/common/constants.ts

```diff
@@ -27,13 +27,16 @@ export const APP_ID = 'maps';
 export const APP_ICON = 'gisApp';
 export const TELEMETRY_TYPE = 'maps-telemetry';
 
-export const MAP_APP_PATH = `app/${APP_ID}`;
+export const MAPS_APP_PATH = `app/${APP_ID}`;
+export const MAP_PATH = 'map';
 export const GIS_API_PATH = `api/${APP_ID}`;
 export const INDEX_SETTINGS_API_PATH = `${GIS_API_PATH}/indexSettings`;
 export const FONTS_API_PATH = `${GIS_API_PATH}/fonts`;
 
-export const MAP_BASE_URL = `/${MAP_APP_PATH}/${MAP_SAVED_OBJECT_TYPE}`;
-
+const MAP_BASE_URL = `/${MAPS_APP_PATH}/${MAP_PATH}`;
+export function createNewMapPath() {
+  return MAP_BASE_URL;
+}
 export function createMapPath(id: string) {
```

Comment:
Seeing these two functions next to each other, they read a little strangely. It might make more sense to name them `getNewMapPath` and `getExistingMapPath` respectively